urls.py

```diff
@@ -0,0 +1,35 @@
+from django.conf import settings
```

Comment:
Are you using urls.py?  This repo seems to be a full Django project, but the urls are all provided by Django or the XBlock workbench.  Usually people just use the workbench directly.